### ISSUE TITLE
Elixir

### ISSUE THREAD

**Issue by abelards** [LeRubyNouveau/liste-de-souhaits#44]
### Les autres langages que les rubyistes aiment bien
Un beau langage créé par José Valim, du sucre syntaxique proche de Ruby sur la plateforme Erlang.
Je voudrais inviter @thbar et @mremond a minima :)

**Comment by nicoolas25** [comment]
Je crois que chez @synbioz ils y mettent de plus en plus le nez, notamment @Bounga.

**Comment by jumichot** [comment]
Intéressé également, j'en fais pas mal depuis quelque mois. On a une partie de notre solution en prod en phoenix / elixir. Je suis pas expert mais je peux donner quelques input sur la transition vers elixir quand on vient de ruby.

**Comment by Bounga** [comment]
Effectivement, j'en fais. D'abord en perso parce que le langage (et la prog fonctionnelle) m'intriguait. Aujourd'hui, on a deux apps en production côté pro, un outil de migration de données (MSSQL -> PG avec beaucoup de données) et une API complète (lecture / écriture / gestion des permissions / upload / etc)

**Comment by thbar** [comment]
Pour ceux que ça intéresse, voilà des notes d'une migration (qui au final n'a pas été finalisée mais pas pour des raisons tech) de Rails à Elixir/Phoenix:

- https://www.youtube.com/watch?v=0OBSv7Se6fQ
- https://speakerdeck.com/thbar/de-rails-a-phoenix-retour-dexperience-sur-une-reecriture-dapplication-saas

Idem je vois qu'il y a pas mal de rubyistes qui se mettent à Elixir.

Un inconvénient est que l'écosystème Elixir, même si il est déjà de bonne taille, reste en dessous de celui de Ruby (cherchez un client IMAP par exemple, c'est pas aussi facile qu'en Ruby), mais je pense tout de même que Elixir peut plaire à beaucoup de Rubyistes.
